server/src/models/template.ts

```diff
@@ -0,0 +1,18 @@
+import mongoose, { Schema } from "mongoose";
```

Comment:
Call this one "arithmeticTemplate" because templates for other types will be different, like areas or fractions